saleor/graphql/order/filters.py

```diff
@@ -55,13 +55,27 @@ def filter_created_range(qs, _, value):
     return qs
 
 
+def filter_order_search(qs, _, value):
+    order_fields = [
+        "discount_name",
+        "token",
```

Comment:
We should also filter by the order number which is definitely more common to use bu customers than the token.